### PR TITLE
Update to output viash 0.9 config

### PR DESCRIPTION
**Pull request by KaiWaldrant**

### CODE REVIEW

File: viash/core/src/project/create_component/script.py [hunk 1]
```diff
@@ -62,59 +61,77 @@ def create_config(par, component_type, pretty_name, script_path) -> str:
...
+      |  FILL IN: A (multi-line) description of how this method works.
       |''')
     if component_type == "method":
       str += strip_margin(f'''\
-        |  # A reference key from the bibtex library at src/common/library.bib (required).
-        |  reference: bibtex_reference_key
+        |# A reference key from the bibtex library at src/common/library.bib (required).
+        |references:
+        |  doi: doi_reference_key
+        |  # Can also be a bibtex: see https://viash.io/versioned/0_9_0/reference/config/references.html#bibtex
```
Comment (rcannood) [inline viash/core/src/project/create_component/script.py:88]: I think it's better to give an example in comments. Also the `library.bib` is now not used anymore :D

```suggestion
        |# references:
        |#   doi: 
        |#     - 10.1000/xx.123456.789
        |#   bibtex:
        |#     - |
        |#       @article{foo,
        |#         title={Foo},
        |#         author={Bar},
        |#         journal={Baz},
        |#         year={2024}
        |#       }
```

File: viash/core/src/project/create_component/script.py [hunk 2]
```diff
@@ -62,59 +61,77 @@ def create_config(par, component_type, pretty_name, script_path) -> str:
...
+      |      # rendering summary tables.
+      |      summary: "FILL IN: A one sentence summary of this metric."
+      |      # A multi-line description of how this component works (required). Used
+      |      # when rendering reference documentation.
+      |      description: |
+      |        FILL IN: A (multi-line) description of how this metric works.
+      |      # A reference key from the bibtex library at src/common/library.bib (required).
+      |      references: 
+      |        doi: doi_reference_key
+      |        # Can also be a bibtex: see https://viash.io/versioned/0_9_0/reference/config/references.html#bibtex
```
Comment (rcannood) [inline viash/core/src/project/create_component/script.py:123]: Same here:

```suggestion
      |      # references:
      |      #   doi: 
      |      #     - 10.1000/xx.123456.789
      |      #   bibtex:
      |      #     - |
      |      #       @article{foo,
      |      #         title={Foo},
      |      #         author={Bar},
      |      #         journal={Baz},
      |      #         year={2024}
      |      #       }
```

File: viash/core/src/project/create_component/script.py [hunk 3]
```diff
@@ -62,59 +61,77 @@ def create_config(par, component_type, pretty_name, script_path) -> str:
...
+      |      summary: "FILL IN: A one sentence summary of this metric."
+      |      # A multi-line description of how this component works (required). Used
+      |      # when rendering reference documentation.
+      |      description: |
+      |        FILL IN: A (multi-line) description of how this metric works.
+      |      # A reference key from the bibtex library at src/common/library.bib (required).
+      |      references: 
+      |        doi: doi_reference_key
+      |        # Can also be a bibtex: see https://viash.io/versioned/0_9_0/reference/config/references.html#bibtex                 
+      |      links
```
Comment (rcannood) [inline viash/core/src/project/create_component/script.py:124]:
```suggestion
      |      links:

**Review by rcannood — CHANGES_REQUESTED** [review]
minor changes, otherwise lgtm!

File: viash/core/src/project/create_component/script.py (hunk 1 above)
Comment (KaiWaldrant) [reply viash/core/src/project/create_component/script.py:88]: I was thinking of doing the same but are the `{}` not going to give errors ?

Comment (rcannood) [reply viash/core/src/project/create_component/script.py:88]: good point! Fixed!

**Pull request merged**